libraries/AP_Compass/Compass_PerMotor.h

```diff
@@ -0,0 +1,69 @@
+/*
+  per-motor compass compensation
+ */
+
+#include <AP_Math/AP_Math.h>
+
+
+class Compass;
+
+// per-motor compass compensation class. Currently tied to quadcopters
+// only, and single magnetometer
```

Comment:
maybe raise this known limitation as an issue for future revisit ?

Reply:
worth discussing on a dev call